e2e/specs/st_file_uploader.spec.js

```diff
@@ -31,27 +31,28 @@ describe("st.file_uploader", () => {
 
   it("shows widget correctly", () => {
     cy.get("[data-testid='stFileUploader']")
-      .first()
+      .eq(0)
       .should("exist");
     cy.get("[data-testid='stFileUploader'] label")
-      .first()
+      .eq(0)
       .should("have.text", "Drop a file:");
 
     cy.get("[data-testid='stFileUploader']")
-      .first()
+      .eq(0)
       .matchImageSnapshot("single_file_uploader");
 
     cy.get("[data-testid='stFileUploader']")
-      .last()
+      .eq(1)
```

Comment:
I made a handful of small changes to the existing file_uploader e2e tests, to make it clearer which one is being selected now that we have 3 different uploaders in the test